src/app/fsat/fsat-warning.service.ts

```diff
@@ -1,20 +1,28 @@
 import { Injectable } from '@angular/core';
-import { FSAT, FanSetup, BaseGasDensity } from '../shared/models/fans';
+import { FSAT, FanSetup, BaseGasDensity, FsatOutput, FsatInput } from '../shared/models/fans';
 import { Settings } from '../shared/models/settings';
 import { ConvertUnitsService } from '../shared/convert-units/convert-units.service';
 import { PsatService } from '../psat/psat.service';
+import { FsatService } from './fsat.service';
 
 @Injectable()
 export class FsatWarningService {
 
-  constructor(private convertUnitsService: ConvertUnitsService, private psatService: PsatService) { }
+  constructor(private convertUnitsService: ConvertUnitsService, private psatService: PsatService, private fsatService: FsatService) { }
 
   checkFieldDataWarnings(fsat: FSAT, settings: Settings): FanFieldDataWarnings {
+    let isMod: boolean;
```

Comment:
have `isModification` be a passed in boolean instead of checking for modifications, doesn't always indicate baseline vs modification